build_runner/lib/src/util/constants.dart

```diff
@@ -6,7 +6,13 @@ import 'dart:io';
 import 'package:crypto/crypto.dart';
 
 /// Relative path to the asset graph from the root package dir.
-final String assetGraphPath = assetGraphPathFor(Platform.script.path);
+final String assetGraphPath = () {
```

Comment:
[optional] Pretty minor but I think the conditional operator improves readability slightly because it draws attention to the thing that is different (toFilePath() vs path) and clearly show the thing that is the same (assetGraphPathFor)

```dart
final String assetGraphPath = () {
  return assetGraphPathFor(Platform.script.scheme == 'file'
      ? Platform.script.toFilePath()
      : Platform.script.path);
}();
```

Reply:
done